src/admin/cms.js

```diff
@@ -136,10 +139,42 @@ Initative.propTypes = {
 	getAsset: PropTypes.object.isRequired
 };
 
+const resourceTags = {resourceTags: globalResourceTags};
+
+const Resources = ({ entry }) => {
+	return <NunjucksPreview
+		entry={entry}
+		path="layouts/resourceDetail.njk"
+		context={({title, focus, source, readability, type, toolPurpose, toolAccessibilityIssues, openSource, link, sharePointUrl, keywords, learnTags, summary, body }) => ({
```

Comment:
It would be ideal to see the field titles in the CMS preview pane:

![image](https://user-images.githubusercontent.com/547933/142129016-a79e950e-a41b-4b94-af55-8e0e1e7b1b0c.png)

Reply:
I'm seeing a different CMS preview based off this branch. All preview related issues you mentioned below cannot be reproduced on my local machine.

<img width="1391" alt="preview" src="https://user-images.githubusercontent.com/572877/142222587-f8b06eff-6463-4199-8716-94b5d25d688c.png">